.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "lib/src/test/resources/ably-common"]
 	path = lib/src/test/resources/ably-common
 	url = https://github.com/ably/ably-common.git
+[submodule "submodules/ably-delta-codec-java"]
```

Comment:
OOI, what is the thinking about this being a submodule?

Reply:
We need this for the delta tests. The library does not refer it directly and has no knowledge of it but the tests have to be able to create the pluggable `vcdiff` decoder and provide it to the library instance.

Reply:
Ok. @paddybyers @QuintinWillison I thought you told me the other day the plan was now to bundle in VCDiff as part of the Java distribution.  Have I got my wires crossed?

Reply:
Good point, though I didn't think we had decided absolutely as we've not yet assessed what the impact is on library size for Android of baking VCDiff in. @codemerx, do you know how much it adds to the size?

Reply:
The jar is about 108K.

Reply:
So, with `ably-android-1.1.6.aar` at just over 340 KB, I think I am right in calculating the impact of 108 KB as an increase of about **32%**.